1.4/python/slim/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright 2017 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM python:3.6-slim
+
+ENV GRPC_PYTHON_VERSION 1.4.0
```

Comment:
Do we need to provide this for 1.4? I'd say keep this for new dockerfiles only

Reply:
I added it for 1.4 to add parity/uniformity for all python images. We can remove the image if necessary